sass/_base.scss

```diff
@@ -343,3 +343,56 @@ a>code {
         }
     }
 }
+
+.call-to-action {
+    background-color: #000;
+    border: 2px solid #000;
```

Comment:
It's actually necessary because some CTAs (the secondary ones) have a special style on hover that keeps the border but inverts the background. If the border wasn't present, the button jumps, making it look glitchy :)